integration/src/main/java/org/exoplatform/task/integration/notification/TaskEditionPlugin.java

```diff
@@ -37,6 +37,6 @@ public String getId() {
   public boolean isValid(NotificationContext ctx) {
     Task task = ctx.value(NotificationUtils.TASK);
     return((task.getAssignee() != null && !task.getAssignee().isEmpty()) ||
-            (task.getCoworker() != null && task.getCoworker().size() > 0)||(task.getWatcher() != null && task.getWatcher().size() > 0));
+            (task.getCoworker() != null && task.getCoworker().size() > 0)||(task.getWatcher() != null && task.getWatcher().size() > 0) || (task.getCreatedBy() != null && task.getCreatedBy() != null));
```

Comment:
There is an unneeded check for nullity for task.getCreatedBy()
```suggestion
            (task.getCoworker() != null && task.getCoworker().size() > 0)||(task.getWatcher() != null && task.getWatcher().size() > 0) || (task.getCreatedBy() != null && task.getCreatedBy() != null));
```